### PR TITLE
Change links in leap_160.yaml to distribution

### PR DESCRIPTION
**Pull request by stacheldrahtje**
Change links in leap_160.yaml to distribution. instead of refering to repositories we now refer to distribution for leap_160

### CODE REVIEW

**Review by imobachgs — COMMENTED** [review]
Thanks for the change. But, please, update the pull request description because it is included in the Git commit message when merging.

**Comment by imobachgs** [comment]
And, please, add an entry in the [changes file](https://github.com/openSUSE/agama/blob/master/products.d/agama-products-opensuse.changes).

File: products.d/leap_160.yaml [hunk 1]
```diff
@@ -30,9 +30,9 @@ software:
       archs: x86_64
     - url: https://download.opensuse.org/repositories/openSUSE:/Leap:/16.0/standard
       archs: aarch64
-    - url: https://download.opensuse.org/repositories/openSUSE:/Leap:/16.0/product/repo/Leap-Packages-16.0-x86_64
+    - url: https://download.opensuse.org/distribution/leap/16.0/repo/oss
       archs: x86_64
-    - url: https://download.opensuse.org/repositories/openSUSE:/Leap:/16.0/product/repo/Leap-Packages-16.0-aarch64
+    - url: https://download.opensuse.org/distribution/leap/16.0/repo/oss
       archs: aarch64
```
[the remark this answers is not in the document]
Comment (jreidinger) [reply products.d/leap_160.yaml:36]: ah, suggestion is created wrong. I mean you can replace all content from line 29

[the remark this answers is not in the document]
Comment (stacheldrahtje) [reply products.d/leap_160.yaml:36]: @lkocman requested to leave standard as they were AFAIK

[the remark this answers is not in the document]
Comment (lkocman) [reply products.d/leap_160.yaml:36]: let's keep standard for now especially in pre-Alpha/Alpha, it's really useful to have it if we need to test a quick fix. Standard is published within 10 minutes, while the product repos take several hours.

**Comment by stacheldrahtje** [comment]
Meh... I put it in PR 1463 the changes to changes

**Comment by imobachgs** [comment]
> Meh... I put it in PR 1463 the changes to changes

Not a problem but, why a separate PR?

File: products.d/leap_160.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jreidinger) [reply products.d/leap_160.yaml:36]: what I commented is not to remove standard but to use more comprehensive format, that use `archs: x86_64,aarch64` for identical entries

**Comment by stacheldrahtje** [comment]
well i'm not fluent in git yet, so I thought there should be an easier way but didn't find it ;-)

**Comment by stacheldrahtje** [comment]
Close as too messy and premature

**Pull request closed without merging**